docs/ceph/ceph_rgw.md

```diff
@@ -677,9 +677,10 @@ Worker Balance
 ## Additional Resources
 
 A screen recording is available [here](https://asciinema.org/a/560091).
+<script id="asciicast-560091" src="https://asciinema.org/a/560091.js" async data-autoplay="true" data-speed="2"></script>
 
-[0]: https://access.redhat.com/articles/1548993
```

Comment:
This is breaking the links rendering for me: the links here are not meant to be shown but in the middle of the sections they're rendered as part of the text.
Can you revert this part?

Reply:
done!